src/rpc/blockchain.cpp

```diff
@@ -985,6 +987,202 @@ static RPCHelpMan gettxoutsetinfo()
     };
 }
 
+static RPCHelpMan dumpcoinstats()
+{
+    return RPCHelpMan{"dumpcoinstats",
+        "Dumps content of coinstats index to a CSV file. The resulting file contains the following columns:\n"
```

Comment:
Could mention that this will have a line for each block?

Reply:
done